src/yarpmanager/src-manager/safe_manager.cpp

```diff
@@ -16,9 +16,12 @@ using namespace yarp::manager;
 #define POST_SEMAPHOR() postSemaphore();
 
 
-SafeManager::SafeManager()
-    :action(MNOTHING), eventReceiver(NULL), busyAction(false)
-{ }
+SafeManager::SafeManager() :
+    m_pConfig(YARP_NULLPTR),
+    action(MNOTHING),
+    eventReceiver(NULL),
```

Comment:
Please use `YARP_NULLPTR` for making easier search and replace when merging `master` in `devel`